Fabric.Identity.API/scripts/tests/identity-cli.tests.ps1

```diff
@@ -71,4 +72,22 @@ describe 'Get-FabricInstallerAccessToken' {
             }
         }
     }
+}
+
+describe 'Functional-Tests' {    
+    BeforeAll {
+        # Set up auth/identity
+        Start-Process "$targetFilePath\..\start-auth-identity.sh" -Wait
```

Comment:
these launch a new process, so this script and the stop-auth-identity script may be failing on the build server which and the test continues regardless.  I imagine there's a way to check the status of the process, will need to look into it